### PR TITLE
Crow improvements and small cleanup.

### PR DESCRIPTION
**Pull request by Shrimpstress**
Implement wing animations and head animation for crows. Move TreasureChestModel to model package. Remove newlines in multiple biome classes.

### CODE REVIEW

**Review by Shnupbups — APPROVED** [review]

**Comment by Shrimpstress** [comment]
Don't merge just yet, I have another small change I'm going to make.

**Comment by Shrimpstress** [comment]
Alright, this should be good once we get more approvals now.

File: src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java]: Unnecessary override?

Comment (Shrimpstress) [reply src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java]: Ah, my bad. I thought that was in the class before the PR, guess I added it myself.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**
